### PR TITLE
✅ Adds tests to answer survey create service

### LINKED ISSUE #435: Add answer_survey.ts tests
We need to add tests to answer_survey.ts to reach 100% coverage of this file.

### PR DESCRIPTION
**Pull request by mizek1**
fix #435 
# Adds tests to answer_survey.ts on services

####
- [ ] **Model testing.**
- [ ] **Request testing.**
- [ ] **System testing.**
- [X] **No tests required.**

### CODE REVIEW

File: frontend/src/api/services/answer_survey.test.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+const mockAnswerSurveyCreate: AnswerSurveyCreate = {
+  survey_id: 1,
+};
+
+describe('AnswerSurveyService', () => {
+  let instance: AnswerSurveyService;
+  let res: AxiosResponse<AnswerSurvey>;
+
+  describe('register method', () => {
+    describe('when returns the expected data', () => {
```
Comment (edimossilva) [inline frontend/src/api/services/answer_survey.test.ts:28]: @mizek1 , this is just a bikeshedding / common mistake among Portuguese speakers, @amiapmorais and I do the same.
```suggestion
    describe('when returns expected data', () => {
```

File: frontend/src/api/services/answer_survey.test.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            }
+          )
+        );
+
+        instance = new AnswerSurveyService(httpClient as any);
+
+        res = await instance.register(mockAnswerSurveyCreate);
+      })
+
+      it('should call the post method once', () => {
```
Comment (edimossilva) [inline frontend/src/api/services/answer_survey.test.ts:45]:
```suggestion
      it('should call post method once', () => {
```

File: frontend/src/api/services/answer_survey.test.ts [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        expect(httpClient.post).toHaveBeenCalled();
+      });
+
+      it('should return the expected data', () => {
```
Comment (edimossilva) [inline frontend/src/api/services/answer_survey.test.ts:49]:
```suggestion
      it('should return expected data', () => {
```

File: frontend/src/api/services/answer_survey.test.ts [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+            }
+          )
+        );
+
+        instance = new AnswerSurveyService(httpClient as any);
+
+        res = await instance.register(mockAnswerSurveyCreate);
+      });
+
+      it('should call the post method', () => {
```
Comment (edimossilva) [inline frontend/src/api/services/answer_survey.test.ts:71]:
```suggestion
      it('should call post method', () => {
```

File: frontend/src/api/services/answer_survey.test.ts [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+        expect(httpClient.post).toHaveBeenCalled();
+      });
+
+      it('should return the expected data', () => {
```
Comment (edimossilva) [inline frontend/src/api/services/answer_survey.test.ts:75]:
```suggestion
      it('should return expected data', () => {
```

**Review by edimossilva — APPROVED** [review]
Nice work!

File: frontend/src/api/services/answer_survey.test.ts (hunk 1 above)
Comment (iurynogueira) [reply frontend/src/api/services/answer_survey.test.ts:28]: I too..

Comment (mizek1) [reply frontend/src/api/services/answer_survey.test.ts]: Yeah! This error is in other test files on the system (which I was inspired to create these).

**Comment by iurynogueira** [comment]
Nice! LGTM.

File: frontend/src/api/services/answer_survey.test.ts (hunk 1 above)
Comment (edimossilva) [reply frontend/src/api/services/answer_survey.test.ts]: This happens a lot, usually I also make it so hope the guys find out lol

**Pull request merged**